src/tests/modules/bson.rs

```diff
@@ -345,3 +345,177 @@ fn debug_print() {
     assert_eq!(format!("{:?}", doc), normal_print);
     assert_eq!(format!("{:#?}", doc), pretty_print);
 }
+
+#[cfg(feature = "uuid-0_8")]
+#[test]
+fn test_binary_constructors() {
+    use crate::bson::UuidRepresentation;
```

Comment:
rather than repeating these `use` statements in each test, we can just add them at the beginning of the module.